packages/augur-ui/src/utils/convert-marketInfo-marketData.ts

```diff
@@ -40,6 +44,7 @@ export function convertMarketInfoToMarketData(
     endTimeFormatted: convertUnixToFormattedDate(marketInfo.endTime),
     creationTimeFormatted: convertUnixToFormattedDate(marketInfo.creationTime),
     categories: marketInfo.categories,
+    isArchived: archivedDuration && (Math.abs(archivedDuration.asDays()) > ARCHIVED_MARKET_LENGTH),
```

Comment:
I think conditional is off. if duration is in whole days then the market could get pruned between day 30 and 31 but the condition would only set isArchived after day 31.

easy fix would be `>=` or use seconds so when 30 days + 1 second elapses we'd set the archived flag.